### DISCUSSION TITLE
Problem with model editor

### DISCUSSION THREAD

**Discussion by Musante** [google/model-viewer#2653] [Q&A]
For some days, in the model-editor uploading and setting the metalness/roughness texture only affects the metalness. 
The model retains the roughness map (or lack of) it had when imported.

Any suggestions?

**Comment by elalish** [comment]
Good catch! #2654

**Reply by Pedergnana** [reply]
@elalish I had the same experience and yap metalness/roughness texture only affects the metalness. I would like to underline also that the ambient occlusion if I use a map RGP where R channel is the AO is not working properly anymore like before the last update
